content/docs/use-cases/fast-data-storage-layer.md

```diff
@@ -0,0 +1,143 @@
+> more details.
+
+[linked]:
+  /doc/user-guide/large-dataset-optimization#file-link-types-for-the-dvc-cache
+
+**You can unify all your data across projects by setting up a [shared cache] as
+primary storage.** This compounds the deduplication and performance benefits.
+Having separated data storage from other aspects of your projects, you can treat
+it as an independent infrastructure layer which can use Ceph,
+[EFS](https://aws.amazon.com/efs/), etc. depending on the access speed and
+frequency required.
+
+![](/img/shared-server.png) _Data storage shared by DVC projects_
+
+<!-- Simplify/ make more general? Not local, symlinks only -->
+
+To balance speed and cost, you can also use a `dvc remote` (or several) as
+secondary storage (e.g. an object store like S3,
+[GCS](https://cloud.google.com/storage/), or Azure Blob Storage). Data only
+needed in older [versions](/doc/use-cases/versioning-data-and-model-files) of
+your project can be kept there. Remote storage is also a great way to back up
+and [share datasets and ML models](/doc/use-cases/sharing-data-and-model-files).
+
+> 🔒 Another benefit of having a clear organization of your near and remote
+> storage locations is to manage user access, as well as centralizing data
+> security policies.
+
+Using DVC to implement a layered data storage system also gives you flexibility
+to redesign the solution in the future without having to change directory
+structures or code.
+
+<details>
+
+## Click to explore a range of advanced implementations
+
+- On one extreme, you can [prepare your data requirements] in a lightweight
+  system without mass storage (0 layers).
+- Connect a primary storage to your local machine or network to transition into
+  a typical (1 layer) working DVC environment.
+- Add a complex remote layer by connecting to multiple cloud storage services,
+  and backup certain data on different ones (2-layer solution). DVC can
+  synchronize them with your <abbr>cache</abbr> when needed.
+- On the other end, you can have a single-layer, remote-only architecture, e.g.
+  for deploying trained ML models to production applications.
+
+[prepare your data requirements]:
+  https://dvc.org/doc/command-reference/add#example-transfer-to-remote-storage
+
+</details>
+
+## Example: Shared Development Server
+
+Some teams prefer using a single shared machine to run their experiments. This
+allows better resource utilization (quick transfers, central storage, GPU
+access, etc.) in a simple way. Each person can work in separate
+<abbr>workspaces</abbr>, and DVC will handle the data linking.
+
+⚙️ Start by configuring a [shared cache].
+
+[shared cache]: /doc/user-guide/how-to/share-a-dvc-cache
+
+> ⚠️ It's not recommended to share a single workspace among users, as it may
+> produce file permission issues.
+
+Let's say you are cleaning up raw data for later stages:
+
+```dvc
+$ dvc add raw
+$ dvc run -n clean_data -d cleanup.py -d raw -o clean \
+          ./cleanup.py raw clean
```

Comment:
I left `run` since it's shorter than `stage add` and then `repro`. Or should I migrate these samples?

Reply:
I think we should move all `dvc run` to `dvc stage add`, and not introduce new uses.  It will probably be gone in 3.0

Reply:
Is it deprecated? Idk

Reply:
It's not officially deprecated but I guess most of us agree it should be, so updated.